src/freedv_interface.h

```diff
@@ -96,6 +99,8 @@ class FreeDVInterface
     std::deque<SRC_STATE*> rateConvObjs_;
     COMP txFreqOffsetPhaseRectObj_;
     std::deque<COMP*> rxFreqOffsetPhaseRectObjs_;
+    struct MODEM_STATS* modemStatsList_;
+    int modemStatsIndex_;
     
```

Comment:
Ok so a separate set of modem stats for each mode?  That's a good idea, as a single global will be overwritten by the last demod your tried, which may not be the one that gets sync :+1: